lib/resolvers/project.js

```diff
@@ -16,5 +20,44 @@ module.exports = ({ models }) => ({ action, data, id }, transaction) => {
           .then(() => project);
       });
   }
+
+  if (action === 'fork') {
+    return ProjectVersion.query(transaction)
+      .where({ projectId: id })
+      .where(builder => builder.whereNotNull('submittedAt').orWhereNotNull('grantedAt'))
+      .orderBy('createdAt', 'desc')
+      .first()
+      .then(version => ProjectVersion.query(transaction).insertAndFetch(pick(version, 'data', 'projectId')));
+  }
+
+  if (action === 'grant') {
+    return ProjectVersion.query(transaction)
+      .where({ projectId: id })
+      .whereNull('grantedAt')
+      .whereNotNull('submittedAt')
+      .orderBy('createdAt', 'desc')
+      .first()
```

Comment:
It might be worth having an actual version parameter that replaces this, so it's an explicit request to grant a particular version.

Reply:
my reasoning here was version at the point of requesting the grant may be different from the actual granted version - if the inspector requests changes a new version is created and this may end up being the granted version.

Reply:
I guess we could get around that by updating the versionId in workflow though which would achieve the same thing

Reply:
Nah, that makes sense. I just haven't fully internalised the workflow here. You're right. I forgot it goes:

* create task
* loads of stuff changes
* resolve task